docs/guides/user/basic-concepts.md

```diff
@@ -92,7 +92,7 @@ To play with timesketch without any installation visit [demo.timesketch.org](htt
 
 There is a dedicated document called [search query guide](search-query-guide.md) to help you create custom searches.
 
-All data within Timesketch is stored in elasticsearch. So the search works similar to ES.
+All data within Timesketch is stored in OpenSearch. So the search works similar to ES.
```

Comment:
s/So the search works similar to ES./So the search works similar to OpenSearch.

Reply:
Fixed